### PR TITLE
Fix english typos and grammar in documentation

### PR DESCRIPTION
**Pull request by arpitjain099**
Documentation can use some typo fixes and grammar fixes in documentation. This PR fixes some of the issues. Please comment if you'd like me to look at more files

### CODE REVIEW

File: uwp/app-resources/compile-resources-manually-with-makepri.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ MakePri.exe is typically used with the `new`, `versioned`, and `resourcepack` op
...
+This warning is displayed when MakePri.exe or MSBuild discovers files or string resources for a given named resource that appear to be marked with language qualifiers, but no candidate is found for a default language. The process for using qualifiers in file and folder names are described in [Tailor your resources for language, scale, and other qualifiers](tailor-resources-lang-scale-contrast.md). A file or folder may have a language name in it, but no resources are discovered that are qualified for the exact default language. For example, if a project uses "en-US" as the default language and has a file named "de/logo.png", but does not have any files that are marked with the default language "en-US", this warning will appear. In order to remove this warning, either file(s) or string resource(s) should be qualified with the default language or the default language should be changed. To change the default language, with your solution open in Visual Studio, open `Package.appxmanifest`. On the Application tab, confirm that the Default language is set appropriately (for example, "en" or "en-US").
```
Comment (Court72) [inline uwp/app-resources/compile-resources-manually-with-makepri.md]:
```suggestion
This warning is displayed when MakePri.exe or MSBuild discovers files or string resources for a given named resource that appear to be marked with language qualifiers, but no candidate is found for a default language. The process for using qualifiers in file and folder names is described in [Tailor your resources for language, scale, and other qualifiers](tailor-resources-lang-scale-contrast.md). A file or folder may have a language name in it, but no resources are discovered that are qualified for the exact default language. For example, if a project uses "en-US" as the default language and has a file named "de/logo.png", but does not have any files that are marked with the default language "en-US", this warning will appear. In order to remove this warning, either file(s) or string resource(s) should be qualified with the default language, or the default language should be changed. To change the default language, with your solution open in Visual Studio, open `Package.appxmanifest`. On the Application tab, confirm that the Default language is set appropriately (for example, "en" or "en-US").
```

File: README.md [hunk 2]
```diff
@@ -14,20 +14,20 @@ You will receive a response like:
...
+Microsoft, Windows, Microsoft Azure, and/or other Microsoft products and services referenced in the documentation
 may be either trademarks or registered trademarks of Microsoft in the United States and/or other countries.
 The licenses for this project do not grant you rights to use any Microsoft names, logos, or trademarks.
 Microsoft's general trademark guidelines can be found at https://go.microsoft.com/fwlink/?LinkID=254653.
 
 Privacy information can be found at https://privacy.microsoft.com/en-us/
 
-Microsoft and any contributors reserve all others rights, whether under their respective copyrights, patents,
-or trademarks, whether by implication, estoppel or otherwise.
+Microsoft and any contributors reserve all other's rights, whether under their respective copyrights, patents,
```
Comment (Court72) [inline README.md]:
```suggestion
Microsoft and any contributors reserve all others rights, whether under their respective copyrights, patents,
```

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 ## Rename Notification
-This repo has recently been renamed from "windows-uwp" to "windows-dev-docs" in order to better represent the content source files contained.
+This repo has recently been renamed from "windows-uwp" to "windows-dev-docs" to better represent the content source files contained.
 
 If you have a copy of the repo on your local machine, you will need to update the associated URL following these steps:
 
 1. `cd` to your local Git directory for the repo and find the remote name with the command: `git remote -v`
 
-You will receive a response like:
+You will receive a response like this:
```
Comment (Court72) [inline README.md]:
```suggestion
You will receive a response like the following example:
```

**Comment by Court72** [comment]
@arpitjain099 - Thanks for the typo fixes!

**Comment by arpitjain099** [comment]
@Court72  I'll submit some more. Thank you for the appreciation

**Pull request merged**